CHANGELOG.md

```diff
@@ -47,15 +49,22 @@ New Features
   feature in twilio-video.js@2.1.0, followed by removing our existing SIP-based
   signaling transport altogether in twilio-video.js@2.2.0.
 
-  **NOTE:** The new signaling transport will reject access tokens containing configuration
-  profiles, which were deprecated when we [announced](https://www.twilio.com/blog/2017/04/programmable-video-peer-to-peer-rooms-ga.html#room-based-access-control)
-  the general availability of twilio-video.js@1.0.0. Use the [Programmable Video REST API](https://www.twilio.com/docs/video/api)
-  if you want to override the default settings while creating a Room.
-
-- This version of twilio-video.js will support Safari 12.1 and above which enables
-  Unified Plan as the default SDP format. We highly recommend that you upgrade your
-  twilio-video.js dependency to this version so that your application continues to
-  work on Safari versions 12.1 and above. (JSDK-2306)
+  **NOTE:** The new signaling transport will:
+  - disconnect you from the Room with an [AccessTokenExpiredError](https://www.twilio.com/docs/api/errors/20104)
+    if while reconnecting, it detects that your AccessToken is expired. Therefore,
+    we recommend that you create an AccessToken that is valid for the maximum
+    allowed session duration, i.e., *4 hours*.
+  - raise an [AccessTokenInvalidError](https://www.twilio.com/docs/api/errors/20101)
+    when you try to join a Room with an AccessToken created using a *non-string*
+    `identity`, instead of a [ParticipantIdentityInvalidError](https://www.twilio.com/docs/api/errors/53200).
```

Comment:
I just got an update on [VIDEO-2009](https://issues.corp.twilio.com/browse/VIDEO-2009) which indicates that video-signaling-gateway-0.5.3 does accept numeric identities. If VSG gets deployed before release of 2.0.0-beta6 then this note should be removed. Alternatively, should we hold the release so that it goes out after the vsg-0.5.3?

Reply:
@ceaglest 
Right now, TCMP is opt-in, and moreover, this release also has support for Safari unified plan which I want to release as soon as possible. I will talk to Yuri about this.

Reply:
@ceaglest I talked to Yuri, and I'm going to remove this section since he is going to deploy VSG v0.5.3 by EOD today.